{
  "uid" : "54856786dd47db58",
  "name" : "Unauthorized export attempt",
  "fullName" : "Certificate Template Security: Unauthorized export attempt",
  "historyId" : "147794cc9fdf1e7d9040ef6ace2be08f",
  "time" : {
    "start" : 1746713381426,
    "stop" : 1746713382007,
    "duration" : 581
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: STEP: Given a certificate template named \"Exportable_Template\" exists\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_102_steps.py\", line 27, in step_given_certificate_template_exists\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: STEP: Given a certificate template named \"Exportable_Template\" exists\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_102_steps.py\", line 27, in step_given_certificate_template_exists\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the admin user is logged into TPC_Web",
      "time" : {
        "start" : 1746713381429,
        "stop" : 1746713381972,
        "duration" : 543
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Given a certificate template named \"Exportable_Template\" exists",
      "time" : {
        "start" : 1746713381972,
        "stop" : 1746713381973,
        "duration" : 1
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: STEP: Given a certificate template named \"Exportable_Template\" exists\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_102_steps.py\", line 27, in step_given_certificate_template_exists\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "When a non-admin user attempts to export the template",
      "time" : {
        "start" : 1746713382007,
        "stop" : 1746713382007,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then export should be denied",
      "time" : {
        "start" : 1746713382007,
        "stop" : 1746713382007,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And the attempt should be logged",
      "time" : {
        "start" : 1746713382007,
        "stop" : 1746713382007,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 5,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Certificate Template Security"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 4,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "ae686d6359aefb09",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/22/#testresult/ae686d6359aefb09",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: Given a certificate template named \"Exportable_Template\" exists\n",
        "time" : {
          "start" : 1746711088889,
          "stop" : 1746711089470,
          "duration" : 581
        }
      }, {
        "uid" : "6aae01f955d81a3d",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/21/#testresult/6aae01f955d81a3d",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: Given a certificate template named \"Exportable_Template\" exists\n",
        "time" : {
          "start" : 1746710254531,
          "stop" : 1746710255105,
          "duration" : 574
        }
      }, {
        "uid" : "502c5c22c94fc850",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/20/#testresult/502c5c22c94fc850",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: Given a certificate template named \"Exportable_Template\" exists\n",
        "time" : {
          "start" : 1746709800539,
          "stop" : 1746709801116,
          "duration" : 577
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "54856786dd47db58.json",
  "parameterValues" : [ ]
}